packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -126,6 +126,13 @@ export const vModelCheckbox: ModelDirective<HTMLInputElement> = {
       }
     })
   },
+  mounted(el, { value }) {
```

Comment:
The fix is likely to happen somewhere else, otherwise, it wouldn't work for SSR

Reply:
> Could you add a test too?

of course.I will do it later.

Reply:
Runtime directives are never used during SSR anyway